<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormatWindow</class>
 <widget class="QDialog" name="FormatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WBFS Format Tool</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QTreeWidget" name="treeWidget">
     <property name="toolTip">
      <string>Select a partition to format</string>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Location</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>GiB</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>WBFS</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Path</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="checkBox_recover">
         <property name="toolTip">
          <string>Format the partition in recovery mode</string>
         </property>
         <property name="text">
          <string>Recover Mode</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QCheckBox" name="checkBox_hss">
           <property name="toolTip">
            <string>Check this box to specify a sector size.  This should be a power of 2.</string>
           </property>
           <property name="text">
            <string>HDD Sector Size</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBoxPowerOf2" name="spinBox_hss">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>Check this box to specify a sector size.  This should be a power of 2.</string>
           </property>
           <property name="minimum">
            <number>512</number>
           </property>
           <property name="maximum">
            <number>4096</number>
           </property>
           <property name="singleStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>512</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="pushButton_format">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Format the selected item with WBFS filesystem</string>
       </property>
       <property name="text">
        <string>Format</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_refresh">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Get the available partitions</string>
       </property>
       <property name="text">
        <string>Refresh List</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_done">
       <property name="text">
        <string>Done</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSpinBoxPowerOf2</class>
   <extends>QSpinBox</extends>
   <header>formatwindow.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>checkBox_hss</sender>
   <signal>clicked(bool)</signal>
   <receiver>spinBox_hss</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>348</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>349</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
